data-sources/apache-traffic-server/config.yml

```diff
@@ -0,0 +1,14 @@
+id: apache-traffic-server
+displayName: Apache Traffic Server
+description: |
+  Optimize your Apache Traffic Server performance using New Relic Apache Traffic Server monitoring.
+icon: logo.png
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration/
+keywords:
+  - apache-traffic-server
+  - apache
+  - ats
+  - NR1_addData
```

Comment:
```suggestion
```
@pkudikyala Can you remove this till the @rahulbasu-nr gap analysis need to complete.

Reply:
Hi @mdumpati,
As per your suggestions, I have removed the keyword. Kindly review it.

Thanks.